cmd/network/packet-capture.go

```diff
@@ -325,3 +366,140 @@ func copyFilesFromPacketCapturePods(o *packetCaptureOptions) error {
 
 	return nil
 }
+
+// desiredPacketCapturePod returns the desired Pod read in from manifests
+func desiredPacketCapturePod(o *packetCaptureOptions, key types.NamespacedName) *corev1.Pod {
```

Comment:
Do you mind removing some of the unnecessary line endings in this function and compacting it?